app/assets/yge/youvegoteth/send.js

```diff
@@ -27,7 +27,7 @@ var unPackAddresses = function() {
       }
     }, 3000);
   }
-  localStorage.setItem('addresses', null);
+  localStorage.setItem('addresses', '');
```

Comment:
This change is required because `localStorage` serializes the value `null` to a string. So, `localStorage.getItem('addresses')` will return a string `'null'` which is truthy in JS. So, the condition in line 22 won't catch this issue.

/cc @mbeacom